app.py

```diff
@@ -38,6 +38,22 @@ def post_random_game_from_users_collection() -> Union[str, Response]:
         return abort(Response(response=str(exc), status=500))
 
 
+@app.route("/all_games")
+def post_all_games_from_users_collection() -> Union[str, Response]:
+    args = request.args
+    user = args.get("user")
+    bgg_companion_api = BggCompanionApi(request_client=RequestsRetryClient())
+    try:
+        board_games = bgg_companion_api.get_users_board_games(user)
+        resp = jsonify(board_games)
+        resp.set_cookie(key=USERNAME_COOKIE_NAME, value=user)
+        return resp
+    except UserIsNoneError as exc:
+        return abort(Response(response=str(exc), status=404))
+    except Exception as exc:
+        return abort(Response(response=str(exc), status=500))
```

Comment:
thinking about error handling more: i don't think we'd want to show the actual exception message to the end user for two reasons. one is that it leaks information about your website (like the fact you use Python and perhaps other details) and two is that the user doesn't have any information to alleviate the issue (because the exception message is helpful for programmers, not end-users.)

perhaps we can have a static string for these exceptions and return those instead of the exception call in the response.

Reply:
Ahhhh yeah this totally makes sense.  I guess the only thing is that right now I've been using the fact that it returns the actual exception message for troubleshooting and development.  So I guess the introduction of logging to the application will be enough to return a static string instead of the actual exception.  

I will create a story for this!

Reply:
https://github.com/JDGiardino/BGG-Companion/issues/57